data/data-dictionary.csv

```diff
@@ -23,10 +23,10 @@ education_type_group,Filter,Apprenticeship,qualification_level,Level 7,,api-only
 education_type_group,Filter,Higher education,qualification_level,Level 7,,api-only
 education_type_group,Filter,Total,qualification_level,Level 8,,api-only
 education_type_group,Filter,Higher education,qualification_level,Level 8,,api-only
-phase_type_grouping,Filter,All schools,,,api-only
-phase_type_grouping,Filter,State-funded primary,,,api-only
-phase_type_grouping,Filter,State-funded secondary,,,api-only
-phase_type_grouping,Filter,State-funded primary and secondary,,,api-only
+phase_type_grouping,Filter,All schools,,,api-only,
+phase_type_grouping,Filter,State-funded primary,,,api-only,
+phase_type_grouping,Filter,State-funded secondary,,,api-only,
```

Comment:
api-only should be the final column, so just shift those line ending commas along before the api-only here. Guessing this was probably done on auto by Excel.